### PR TITLE
fix: Don't update `UNIQUE` columns (DuckDB contraint)

### PR DESCRIPTION
**Pull request by candiduslynx**

### CODE REVIEW

File: plugins/destination/duckdb/go.mod [hunk 1]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/cloudquery/plugin-pb-go v1.0.9
 	github.com/cloudquery/plugin-sdk/v3 v3.10.4
 	github.com/google/uuid v1.3.0
-	github.com/marcboeker/go-duckdb v1.3.0
+	github.com/marcboeker/go-duckdb v1.4.0
```
Comment (candiduslynx) [inline plugins/destination/duckdb/go.mod:11]: rebuilt static libs at least

File: plugins/destination/duckdb/client/write.go [hunk 2]
```diff
@@ -164,15 +152,61 @@ func (c *Client) WriteTableBatch(ctx context.Context, table *schema.Table, recor
...
+	if err != nil {
+		return "", err
+	}
+	defer f.Close() // we don't care here, as the happy-path will actually check the error
+	fileName = f.Name()
+
+	// prep file writer
+	fw, err := pqarrow.NewFileWriter(sc, f,
+		parquet.NewWriterProperties(
+			parquet.WithVersion(parquet.V2_4),
```
Comment (disq) [inline plugins/destination/duckdb/client/write.go:169]: why not `V2_LATEST` (which is `V2_6`?)

Comment (candiduslynx) [reply plugins/destination/duckdb/client/write.go]: 8e634aca3bce98b3c0c7b3c8135b38d9a5710669

**Review by erezrokah — COMMENTED** [review]
Thanks for the PR @candiduslynx, do you mind updating the title to reflect what this change fixes so it shows up in the change log?

**Review by erezrokah — APPROVED** [review]

**Pull request merged**